### PR TITLE
Add openssl support to Windows build environment

### PR DESCRIPTION
**Pull request by gtritchie**
**Requires rerunning `install-dependencies` and `cmake`.**

This seeds openssl for Windows for other ongoing work not included in this PR, I have confirmed it is working as expected in that other branch.

The openssl package was built via cross-compilation using MinGW on Linux. See README inside the downloaded package for full details.

### CODE REVIEW

**Comment by jmcphers** [comment]
Do we need to add this to `NOTICE`?

File: src/cpp/core/system/Crypto.cpp [hunk 1]
```diff
@@ -70,26 +70,26 @@ Error lastCryptoError(const ErrorLocation& location)
                          "lastCrytpoError called with no pending error",
                          location);
    }
-
```
Comment (kevinushey) [inline src/cpp/core/system/Crypto.cpp]: Can we try to avoid making whitespace-only changes in the diffs?

**Comment by gtritchie** [comment]
Sure, guess I’ve got editor set to automatically clean up stray whitespace. You can add ?w=1 to github url to hide it this time around.

**Comment by gtritchie** [comment]
I’ll take a look but shouldn’t it already be there since we use it on server already?

**Comment by jmcphers** [comment]
We do have openssl in NOTICE already, just double-check that what you're adding has exactly the same license/terms as what exists today.

**Comment by gtritchie** [comment]
Cleaned up whitespace changes and updated NOTICE. Should be good to go.

**Pull request merged**